### PR TITLE
DOCSP-30796 Update Views UI Information

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION
Updated steps across the page to reflect accurate, updated UI.

## STAGING
https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-30796-compass-ui-views/views/

## JIRA
https://jira.mongodb.org/browse/DOCSP-30796

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64b6fa4b87b3b360469f7831

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/views.txt [hunk 1]
```diff
@@ -125,12 +123,13 @@ Modify the Source of a View
...
-1. Click the desired view in the left navigation.
+1. Hover over the desired view in the left navigation.
+
+#. Click the appearing :guilabel:`Ellipses (...)` button.
 
-#. At the top of the view, click :guilabel:`Modify Source`. This
-   button opens the
-   :ref:`aggregation pipeline builder <compass-agg-builder>` and
-   populates the pipeline used to create the view.
+#. In the dropdown, click :guilabel:`Modify view`. This
```
Comment (sarah-olson-mongodb) [inline source/views.txt:130]: ([terminology](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#d))
There are a few ways you could change this. But probably "In the drop-down menu..."

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks so much @ajhuh-mdb. LGTM after one minor thing.

File: source/views.txt (hunk 1 above)
Comment (ajhuh-mdb) [reply source/views.txt:130]: Updated!

File: source/views.txt [hunk 2]
```diff
@@ -82,16 +80,16 @@ To open a view, either:
...
 
-- :doc:`/documents`
-- :ref:`compass-agg-builder` to aggregate based on your view
-- :doc:`/query-plan`
-- :doc:`/indexes`
-- :doc:`/schema`
-- :doc:`/validation`
+- :ref:`Document management <compass-documents>`
+- :ref:`Aggregation Pipeline Builder <compass-agg-builder>`
+- :ref:`Explain plans <explain-plans>`
```
Comment (himanshusinghs) [inline source/views.txt:89]: The current version of Compass (v1.39) do not have Explain tab anymore. We moved the entire functionality inside a button added to the query bar itself in v1.38.1. If this info still applies to this change (I guess it does) then we might be better without this list item.

**Review by himanshusinghs — APPROVED** [review]
Noticed one small thing otherwise it looks good.

File: source/views.txt (hunk 2 above)
Comment (ajhuh-mdb) [reply source/views.txt:89]: Noted and updated!

**Pull request merged**